### DISCUSSION TITLE
Is there a way to rebase a specific PR with the `renovate` cli?

### DISCUSSION THREAD

**Discussion by crushjz** [renovatebot/renovate#21378] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

34

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### If you're self-hosting Renovate, tell us what version of the platform you run.

15

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

When manually merging multiple MRs in the same repository, there is an high chance of merge conflicts on the npm/pnpm/yarn lockfile.

Is there a way using the `renovate` CLI to run the rebase on a specific PR instead on the whole repository?

### What you tried so far.

I'm able to run `renovate [repository-name]` on a single repository. But I'd like to run it on a specific PR.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by viceice** [comment, marked as answer]
not possible

**Reply by crushjz** [reply]
Thanks for the answer 🙏
